flow/project.py

```diff
@@ -3070,24 +3488,45 @@ def labels(self, job):
 
         See also: :meth:`~.label`
         """
+        aggregates = []
```

Comment:
Should we ask for the labels of a job now or a group of jobs? I am inclined to think the later, aggregate labels shouldn't show up for single jobs.

Reply:
Since labels will be having an aggregator function associated with it, hence the implementation that I'll be going forward with will also incline towards te latter part.